libraries/src/Layout/FileLayout.php

```diff
@@ -238,7 +238,7 @@ public function addIncludePath($path)
 	/**
 	 * Add one or more paths to include in layout search
 	 *
-	 * @param   string  $paths  The path or array of paths to search for layouts
+	 * @param   string|array  $paths  The path or array of paths to search for layouts
```

Comment:
```suggestion
	 * @param   string|string[]  $paths  The path or array of paths to search for layouts
```

Document that it is expected to be an array of strings instead of just a generic array.  It provides more clarity in the documentation (both in the IDE and when it is parsed to an HTML reference, see https://api.joomla.org/cms-3/classes/Joomla.CMS.HTML.HTMLHelper.html#property_includePaths as an example of where a property is documented as `string[]`) and for developers to reference.

Reply:
fixed by: https://github.com/joomla/joomla-cms/pull/29965 Thanks